cordapp/src/main/kotlin/net/corda/plugins/SigningOptions.kt

```diff
@@ -2,207 +2,160 @@ package net.corda.plugins
         private const val DEFAULT_STORETYPE = "JKS"
         private const val DEFAULT_KEYPASS = "cordacadevkeypass"
         const val DEFAULT_KEYSTORE = "certificates/cordadevcakeys.jks"
         const val DEFAULT_KEYSTORE_FILE = "cordadevcakeys"
         const val DEFAULT_KEYSTORE_EXTENSION = "jks"
+        const val SYSTEM_PROPERTY_PREFIX = "signing."
     }
 
-    // Methods accept null value, this allows to use e.g. 'alias System.getProperty('key')' in Gradle configuration file,
-    // instead of requiring 'alias System.getProperty('key','')'
-    @get:Input
-    var alias = ""
-
-    fun alias(value: String?) {
-        alias = value ?: ""
+    /** Option keys for ANT task. */
+    class Key {
+        companion object {
+            val JAR = "jar"
+            val ALIAS = "alias"
+            val STOREPASS = "storepass"
+            val KEYSTORE = "keystore"
+            val STORETYPE = "storetype"
+            val KEYPASS = "keypass"
+            val SIGFILE = "sigfile"
+            val SIGNEDJAR = "signedjar"
+            val VERBOSE = "verbose"
+            val STRICT = "strict"
+            val INTERNALSF = "internalsf"
+            val SECTIONSONLY = "sectionsonly"
+            val LAZY = "lazy"
+            val MAXMEMORY = "maxmemory"
+            val PRESERVELASTMODIFIED = "preservelastmodified"
+            val TSACERT = "tsaurl"
+            val TSAURL = "tsacert"
+            val TSAPROXYHOST = "tsaproxyhost"
+            val TSAPROXYPORT = "tsaproxyport"
+            val EXECUTABLE = "executable"
+            val FORCE = "force"
+            val SIGALG = "sigalg"
+            val DIGESTALG = "digestalg"
+            val TSADIGESTALG = "tsadigestalg"
+        }
     }
 
     @get:Input
-    var storepass = ""
-
-    fun storepass(value: String?) {
-        storepass = value ?: ""
-    }
+    var alias = System.getProperty(SYSTEM_PROPERTY_PREFIX + Key.ALIAS, DEFAULT_ALIAS)
 
     @get:Input
-    var keystore = ""
-
-    fun keystore(value: String?) {
-        keystore = value ?: ""
-    }
+    var storepass = System.getProperty(SYSTEM_PROPERTY_PREFIX + Key.STOREPASS, DEFAULT_STOREPASS)
 
     @get:Input
-    var storetype = ""
-
-    fun storetype(value: String?) {
-        storetype = value ?: ""
-    }
+    var keystore = System.getProperty(SYSTEM_PROPERTY_PREFIX + Key.KEYSTORE, DEFAULT_KEYSTORE)
 
     @get:Input
-    var keypass = ""
+    var storetype = System.getProperty(SYSTEM_PROPERTY_PREFIX + Key.STORETYPE, DEFAULT_STORETYPE)
 
-    fun keypass(value: String?) {
-        keypass = value ?: ""
-    }
+    @get:Input
+    var keypass = System.getProperty(SYSTEM_PROPERTY_PREFIX + Key.KEYPASS, DEFAULT_KEYPASS)
 
     @get:Input
     var sigfile = ""
 
-    fun sigfile(value: String?) {
-        sigfile = value ?: ""
-    }
-
     @get:Input
     var signedjar = ""
 
-    fun signedjar(value: String?) {
-        signedjar = value ?: ""
-    }
-
     @get:Input
     var verbose = ""
```

Comment:
I don't think `verbose` is an `@Input`. Probably more likely it should be `@Console`.